pkg/util/cli/deprecated.go

```diff
@@ -0,0 +1,81 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package cli
```

Comment:
We should add comment on exported method and types.

Reply:
Indeed! Thank you for catching that.